### ISSUE TITLE
Add a notice that all tabs will be closed when clicking the addon button for the first time.

### ISSUE THREAD

**Issue by YousufSSyed** [tabwrangler/tabwrangler#418]
**Is your feature request related to a problem? Please describe.**
I wanted to take a look at the addon before deciding if I want to use it. I knew that the tab would close a bunch of tabs, when I first clicked the button I thought it would show a page for dialogue rather than close all my tabs right away. That's really frustrating, but to make things worse, I uninstalled and reinstalled the addon to reproduce this, **and it closed most of my tabs without even clicking anything!**

**Describe the solution you'd like**
The addon should show some sort of splash screen when first installing the extension about what it does, and have the user confirm it will close a bunch of tabs the first time

**Comment by ssorallen** [comment]
I agree, this sounds like a positive improvement. I would implement it as follows:

1. Initial state of Tab Wrangler after installation is "Paused"
2. TW has a different icon, maybe exclamation point to draw attention
3. Popup has initial info and "Start wrangling" button or similar to start the extension for the first time

**Comment by YousufSSyed** [comment]
Yeah that sounds like a good implementation.
